sonic-utilities-tests/mock_tables/dbconnector.py

```diff
@@ -41,7 +42,11 @@ def connect_SonicV2Connector(self, db_name, retry_on=True):
     # add the namespace to kwargs for testing multi asic
     self.dbintf.redis_kwargs['namespace'] = self.namespace
     # Mock DB filename for unit-test
-    self.dbintf.redis_kwargs['db_name'] = db_name
+    global dedicated_dbs
+    if dedicated_dbs and dedicated_dbs.get(db_name):
+        self.dbintf.redis_kwargs['db_name'] = dedicated_dbs[db_name]
```

Comment:
Looks like the mocked values will not get set in db. Check L94 where the path is constructed

Reply:
Thanks.
Actually, this is because the vstest runs under `sonic-utilities/deb_dist/sonic-utilities-1.2/sonic-utilities-tests` folder while I verified all the test cases in `sonic-utilities/sonic-utilities-tests` folder. Any newly added test files need to be added to `sonic-utilities/setup.py` so that they can be copied into `sonic-utilities/deb_dist/sonic-utilities-1.2/sonic-utilities-tests` folder and then the test can pass.
Fixed.